README.md

```diff
@@ -75,9 +75,12 @@ NOTE:
 | CephFS | Dynamically provision, de-provision File mode RWO volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=14.2.2)  | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode RWX volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode ROX volume  | Alpha          | >= v3.0.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
-|        | Creating and deleting snapshot                            | -              | -                  | -                | -                    | -                  |
-|        | Provision volume from snapshot                            | -              | -                  | -                | -                    | -                  |
-|        | Provision volume from another volume                      | -              | -                  | -                | -                    | -                  |
+|        | Creating and deleting snapshot                            | Alpha          | >= v3.1.0          |  >= v1.0.0       | Octopus (>=v15.2.3)  | >= v1.17
+0         |
```

Comment:
Is the `0` here intentional?

Reply:
Seems to be coming in the `plugin` section

Reply:
@Yuggupta27 Thanks for catching this @humblec cephfs matrix is broken https://github.com/ceph/ceph-csi/blob/68b56e260f98fb4ba495d090e23780d50c18875b/README.md#support-matrix

Reply:
Thanks @Yuggupta27 . corrected it :+1:

Reply:
still, I see the same issue

Reply:
@Madhu-1 can you referesh